env.md

```diff
@@ -0,0 +1,118 @@
+# Inventory of Agoric SDK Environment Variables
+
+```js
+process.env.DOCUMENTATION // for power users
+```
+
+TODO: Fill out more details!
+
+This file describes environment variables that influence the execution of the
+Agoric SDK.  They are documented here as an advanced feature to be kept
```

Comment:
I suggest striking this 2nd sentence about up-to-date with the source code. That seems implicit in all our docs. We have a big Beta thingy at the top, after all.